node_modules/oae-tenants/lib/model.js

```diff
@@ -37,6 +38,7 @@ var Tenant = module.exports.Tenant = function(alias, displayName, host, opts) {
     that.countryCode = opts.countryCode;
     that.active = (opts.active !== false);
     that.isGlobalAdminServer = (opts.isGlobalAdminServer === true);
```

Comment:
We probably don't need to pass in `opts.isGlobalAdminServer` and `opts.isGuestTenant` here, as we can just infer that from the config file.